packages/components/title/stories/index.stories.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import type { StoryFn, Meta } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import Title, { type TitleProps } from '..';
+import '../style.scss';
+
+export default {
+	title: 'Block Components/Title',
+	component: Title,
+	argTypes: {
+		className: {
+			control: 'text',
+			table: {
+				type: {
+					summary: 'string',
+				},
+			},
+			description:
+				'Additional CSS classes to apply to the title element.',
+		},
+		headingLevel: {
+			control: 'select',
+			options: [ '1', '2', '3', '4', '5', '6' ],
+			table: {
+				type: {
+					summary: "'1' | '2' | '3' | '4' | '5' | '6'",
+				},
+			},
```

Comment:
Doesn't look like this matter visually right?

Reply:
Correct, that's just what h tag is used